library/certificate_request.py

```diff
@@ -26,7 +26,31 @@
     description:
       - Domain (or list of domains) to be included in the
         certificate. Also can provide the default value for
-        common_name.
+        I(common_name).
+    required: false
+  ip:
+    description:
+      - IP (or list of IPs) to be included in the certificate.
+        IPs can be IPv4, IPv6 or both. Also can provide the
+        default value for I(common_name).
+    required: false
+  email:
+    description:
+      - Email (or list of emails) to be included in the
+        certificate. Also can provide the default value for
+        I(common_name).
+    required: false
+  owner:
+    description:
+      - User name (or user id) for the certificate and key files.
+    required: false
```

Comment:
Is there a default value for `owner` and `group` e.g. `UMASK` or such ?

Reply:
Not really. I was leaving that to Certmonger default.